public/assets/js/index.js

```diff
@@ -5,50 +5,129 @@
  */
 
 // Prints out selected values for drop down menu
-$(document).ready(function () { 
+$(document).ready(function () {
     // Global variables
     var canvas = document.getElementById("myCanvas");
-    var ctx = canvas.getContext("2d"); 
-    var selected = $('#option').val();
+    var ctx = canvas.getContext("2d");
+    var dropdown = document.getElementById("option");
+    var selected = null;
+    var setId;
+    var accessoryId;
+    var accessories = [];
     var images = [];
     var imageCount = 0;
-    var optionSelected = 0;
     var car;
     var frontWheel;
     var backWheel;
     var spoiler;
-    var storage;    
-    var empty = true;
-    
-    (function() {
-        console.log(selected);
-        
-        initialize();
-        
+    var storage;
+    var initialLoad = true;
+    var done = false;
+    var changed = false;
+
+    (function () {
+        // First ajax request
+        ajaxRequest("../info/bundle/");
+        // Waits for ajax to complete before continuing
+        waitForAjax();
+
         // Constantly check the selection in drop down
-        $("#option").change(function(event) {
-            empty = true;
-            selected = $('#option').val();
-             if (selected == "car1") {
-                 console.log(selected);
-                 optionSelected = 0;
-                 checkLoaded();
-             } else if (selected == "car2") {
-                 console.log(selected);
-                 optionSelected = 1;
-                 checkLoaded();
-             } else {
-                 console.log(selected);
-                 optionSelected = 2;
-                 checkLoaded();
-             }
+        $("#option").change(function (event) {
+            setId = $('#option').val();
+            if (setId == 1) {
```

Comment:
why can't we just pass the setId in one statement

Reply:
That works too.

Reply:
I guess just call ajaxRequest("../info/bundleDetail/" + setId); waitForAjax(); instead of doing the checks.